server/db/schemas/test_results.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE test_results(
+    test_id SERIAL PRIMARY KEY,
+    patient_id INT NOT NULL,
+    test_result VARCHAR(255) NOT NULL,
+    type_of_test VARCHAR(255) NOT NULL,
+    Date_of_test TIMESTAMP NOT NULL,
```

Comment:
Lower case first letter and add `WITH TIME ZONE` so it follows the standard ISO date format

```suggestion
    date_of_test TIMESTAMP WITH TIME ZONE NOT NULL,
```